### PR TITLE
Add @tiicotaco

### PR DESCRIPTION
**Pull request by tiicotaco**

### CODE REVIEW

File: site/class-2-debug/tiicotaco/index.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+body {
+  margin: 0;
+}
+
+text{
+    fill: black;
+}
+
+svg {
+  width: 100w;
```
Comment (wooorm) [inline site/class-2-debug/tiicotaco/index.css]: w -> vw

File: site/class-2-debug/tiicotaco/index.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  height: 100h;
```
Comment (wooorm) [inline site/class-2-debug/tiicotaco/index.css]: h -> vh

File: site/class-2-debug/tiicotaco/index.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.axis path {
+  fill: no;
```
Comment (wooorm) [inline site/class-2-debug/tiicotaco/index.css]: no -> none

File: site/class-2-debug/tiicotaco/index.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<!--Based on https://bl.ocks.org/mbostock/3887118 by Mike Bostock-->
+<meta charset=utf8>
+<!--Prevent favicon error from showing up-->
+<link rel="shortcut icon" type=image/x-icon href=data:image/x-icon;,>
+<title>@tiicotaco</title>
+<meta content=width=device-width,initial-scale=1 name=viewport>
+<link rel=stylesheet href=../index.css>
```
Comment (wooorm) [inline site/class-2-debug/tiicotaco/index.html]: ../index.css -> index.css

File: site/class-2-debug/tiicotaco/index.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+<svg></svg>
+<script src=https://d3js.org/d3.v4.min.js></script>
+<script src=../index.js></script>
```
Comment (wooorm) [inline site/class-2-debug/tiicotaco/index.html]: ../index.js -> index.js

**Comment by wooorm** [comment]
Your work is late and I unfortunately don’t have to time to review student work passed the due date.

Feel free to upload more new work, or this work, if it’s uploaded in the right place and correct though, you won’t get points for it, but I’d of course love to see it!

**Comment by tiicotaco** [comment]
Hi,

Before class started I tried submitting my assessment, but I made a mistake and couldn’t figure out what I was doing wrong. I missed the deadline and then decided to ask the teacher during class to see what I was doing wrong, so after the lecture I asked Laurens if he could help me out and together we figured out what I was doing wrong. I know this is my own responsibility and understand if I still don’t get any grade, just wanted to clarify the subject.

Thanks,

Tico van Beurden
ID2

**Comment by wooorm** [comment]
@tiicotaco You’re commenting here on GitHub on your debug assignment, which was on time but broken: I asked you to fix it, but you didn’t in the last 3 days, so I’m regarding it as late now.

On your question: Laurens is responsible for grading your assessment; I’m not interfering in any way with what you agreed on.

**Comment by tiicotaco** [comment]
Awkward… Totally forgot about it, sorry and thanks for clarifying!

Kind regards,

Tico van Beurden

**Comment by wooorm** [comment]
No problem, kinds regards! 😉

**Pull request closed without merging**